frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckUserExistsHandler.java

```diff
@@ -150,6 +156,9 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
             AuditableEvent auditableEvent;
             var rpPairwiseId = AuditService.UNKNOWN;
             var userMfaDetail = new UserMfaDetail();
+            var session = userContext.getSession();
+            var sessionId = session.getSessionId();
+            var maybeAuthSession = authSessionService.getSession(sessionId);
```

Comment:
I was imagining the session id as a final field we'd need to address, but yeah makes perfect sense to do that.

Reply:
Hmm, I'm not sure about this. I think the account interventions lambda is generally next and it only uses this for auditing, so can fail gracefully, then it the login lambda it would get set anyway, but that will fail with a missing auth session, so we are just deferring the failure.